src/rbac.rs

```diff
@@ -164,7 +173,7 @@ mod tests {
     }
 
     #[test]
-    #[should_panic = "Unauthorized"]
+    #[should_panic = "Unauthorized role"]
```

Comment:
@nearken I don't know about in Rust, but in tests in most languages I like to have the constants used here too so that the tests aren't brittle.

Reply:
I agree, but it didn't work ! Errors were being thrown

Reply:
I think it is safer to have tests use hardcoded values anyways.